include/oneapi/dpl/internal/inclusive_scan_by_segment_impl.h

```diff
@@ -117,7 +98,7 @@ inclusive_scan_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIter
     transform_inclusive_scan(policy1, make_zip_iterator(first2, _mask.get()),
                              make_zip_iterator(first2, _mask.get()) + n, make_zip_iterator(result, _mask.get()),
                              internal::segmented_scan_fun<ValueType, FlagType, BinaryOperator>(binary_op),
-                             oneapi::dpl::__internal::__no_op(), ::std::make_tuple(initial_value, initial_mask));
+                             oneapi::dpl::__internal::__no_op());
```

Comment:
Why the initial_mask doesn't need any more? Are sure the implementation remains correct?

Reply:
Yes,  the algo's implementation remains correct.